designs/setdict-design.md

```diff
@@ -0,0 +1,152 @@
+# Set/Tuple/Dictionary: Project Design
+
+## Week 7 Update
+
+### Passed Test Cases:

```

Comment:
I'm confused why these tests are completely different from the test cases below. Why wouldn't you try passing the tests you proposed?

Reply:
The main reason was that we hadn't implemented a print_set() function back then. Therefore we had to settle with using len() and "in" to help us with the testing. We will fix it.